protocol/signalfx/signalfxlistener.go

```diff
@@ -334,6 +334,14 @@ func SetupJSONByPaths(r *mux.Router, handler http.Handler, endpoint string) {
 	r.Path(endpoint).Methods("POST").Handler(handler)
 }
 
+// SetupJSONByNPaths tells the router which paths the given handler (which should handle the given
+// endpoint) should see
```

Comment:
my annoying nit of always :) punctuation... now ducking for cover 😛

Reply:
Sorry for missing it after you commented. Will fix next time. We should add a linter to enforce it.